duckduckgo/Android
# Added Let's Encrypt ISRG Root X1 and ISRG Root X2 root certificates to the Android system cert trust store

Task/Issue URL: https://app.asana.com/0/72649045549333/1199564873039373/f
Tech Design URL: 
CC: 

**Description**:
* Bundled ISRG Root X1 and ISRG Root X2 root certificates with the binary APK
* Modified network-security-config XML file to add ISRG Root X1/X2 to the system certificate trust store

**Steps to test this PR**:
On device with API = 24
1. Install app from develop
2. Navigate to https://valid-isrgrootx1.letsencrypt.org/
3. Verify the page does not load
4. Install app from this PR
5. Navigate to https://valid-isrgrootx1.letsencrypt.org
6. Verify the page does load


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
